packages/apps/esm-login-app/src/footer.component.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { interpolateUrl, useConfig } from '@openmrs/esm-framework';
+import { useTranslation } from 'react-i18next';
+import { type ConfigSchema } from './config-schema';
+import styles from './login/login.scss';
+
+const Footer: React.FC = () => {
+  const {t} = useTranslation();
+  const config = useConfig<ConfigSchema>();
+  const logos = config.footer.logos || [];
+
+  return (
+    <div className={styles.footer}>
+      <p className={styles.poweredByTxt}>{t('poweredBy', 'Powered by')}</p>
+      <div className={styles.logosContainer}>
+        <svg role="img" className={styles.poweredByLogo}>
+          <use href="#omrs-logo-full-color"></use>
```

Comment:
```suggestion
          <use href="#omrs-logo-partial-mono"></use>
```

This change should not cause the RefApp defaults to diverge from the designs.